docs/src/capi/explanation/in-place-upgrades.md

```diff
@@ -0,0 +1,236 @@
+# In-Place Upgrades
+
+Upgrading the Kubernetes version of the machines is sometimes a necessity. 
+Rolling upgrades are the most popular way to go, but in certain situations 
+we might want or need to go with an in-place upgrade. Examples of these 
+situations are:
```

Comment:
```suggestion
Regularly upgrading the Kubernetes version of the machines in a cluster is important. 
While rolling upgrades a popular strategy, certain situations 
may demand in-place upgrades:
```
Try and focus your text more, as you write something ask yourself about each piece: is this valuable information or just nice fluff. The art here is to convey the minimum information in a way that still flows nicely.

As a second note, make statements that are confident and avoid `sometimes` and `might want ` etc.